streams/src/test/java/org/apache/kafka/streams/state/internals/metrics/RocksDBMetricsRecorderGaugesTest.java

```diff
@@ -211,14 +215,19 @@ private void runAndVerifySumOfProperties(final String propertyName) throws Excep
 
         final long recordedValue1 = 5L;
         final long recordedValue2 = 3L;
-        when(dbToAdd1.getAggregatedLongProperty(ROCKSDB_PROPERTIES_PREFIX + propertyName)).thenReturn(recordedValue1);
-        when(dbToAdd2.getAggregatedLongProperty(ROCKSDB_PROPERTIES_PREFIX + propertyName)).thenReturn(recordedValue2);
+        if (aggregated) {
+            when(dbToAdd1.getAggregatedLongProperty(ROCKSDB_PROPERTIES_PREFIX + propertyName)).thenReturn(recordedValue1);
+            when(dbToAdd2.getAggregatedLongProperty(ROCKSDB_PROPERTIES_PREFIX + propertyName)).thenReturn(recordedValue2);
+        } else {
+            when(dbToAdd1.getLongProperty(ROCKSDB_PROPERTIES_PREFIX + propertyName)).thenReturn(recordedValue1);
+            when(dbToAdd2.getLongProperty(ROCKSDB_PROPERTIES_PREFIX + propertyName)).thenReturn(recordedValue2);
+        }
```

Comment:
I would move this part into `runAndVerifyBlockCacheMetricsWithMultipleCaches()`. Before we forwarded the call to this because the verifications were basically the same. That changed now, so I think it is cleaner to separate both.

Reply:
Done.